configurable-rules/azure-apim-unsupported-keyword/openapi.yaml

```
@@ -0,0 +1,89 @@
+openapi: 3.1.0
+externalDocs: 
+  url: example.com
+  
+info: 
+  title: Azure APIM test case.
+  summary: Azure APIM test
+  version: 1.0.0
+  
+security: []
```

I suggest leaving comments like the one below so that it's clear what will be highlighted even without running the linter:

```suggestion
security: [] # violates rule/security-ignored-azure-apim, should not be defined
```